stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -13,9 +11,10 @@ Although the maximum amount is 7 for the icons or images, it is highly recommend
 
 ##Usage
 Use the rating indicator in forms, tables, or in a dialog box.
-        `
-    },
-    tags: ['f3', 'a11y', 'theme']
+        `,
+        tags: ['f3', 'a11y', 'theme'],
+        components: ['rating-indicator']
+    }
```

Comment:
Note: `tags` and `components` belongs _inside_ `parameter` block or else it won't work